lib/core/remove-file.ts

```diff
@@ -0,0 +1,26 @@
+import fs from "fs";
+
+import { alerts } from "./alerts";
+import { getTypeDefinitionPath } from "../typescript";
+
+/**
+ * Given a single file remove the generated types if they exist
+ *
+ * @param file the SCSS file to generate types for
+ */
+export const removeFile = (file: string): void => {
+  if (fs.existsSync(file)) {
```

Comment:
If I'm reading this correctly, does the `unlink` event mean the file was removed? If so, won't this always return true and always early return since it's invoked in `unlink`?

Reply:
The `unlink` event means that the file has been removed, so this will always be false for when the file is a `*.scss` file. I think I should rename this to `removeSCSSTypeDefinitionFile` since it will only remove the typed file and is not a general remove file function. So the first check makes sure the `*.scss` file doesn't exist and then removes the associated type definition file.

Reply:
Hmm, I'm a bit confused. The `unlink` event is triggered for the `.scss` file, which is the `file` we pass to `removeSCSSTypeDefinitionFile `, which first checks if that file exists (which it doesn't, since the `unlink` event means it was removed)? Am I misunderstanding how that works?

Reply:
Yep that is a correct understanding. I will remove the branch.